lua/toggleterm.lua

```diff
@@ -352,22 +352,23 @@ local function setup_autocommands(_)
     end
 
     -- Extra mouse fix for tmux
-    -- If tmux mouse mode is enabled
-    local output = vim.fn.system 'tmux display -p "#{mouse}"'
-    if output:sub(1, 1) == "1" then
-      -- Disable tmux mouse while using toggleterm
-      api.nvim_create_autocmd({ "TermEnter", "WinEnter <buffer>" }, {
-        desc = "Disable tmux mouse while using toggleterm",
-        group = AUGROUP,
-        callback = function() vim.fn.system "tmux set mouse off" end,
-      })
-
-      -- Enable tmux mouse when mouse leaves toggleterm
-      api.nvim_create_autocmd({ "WinLeave <buffer>" }, {
-        desc = "Enable tmux mouse when mouse leaves toggleterm",
-        group = AUGROUP,
-        callback = function() vim.fn.system "tmux set mouse on" end,
-      })
+    if os.getenv "TMUX" ~= nil then
```

Comment:
```suggestion
    if vim.env.TMUX then
```
Should be enough for this